docs/pages/management/admin/labels.mdx

```diff
@@ -280,6 +282,58 @@ computer. Your Teleport user must be authorized to access the server.
    ip-192-168-13-57 ⟵ Tunnel       arch=x86_64,environment=dev,hostname=ip-192-168-13-57
    ```
 
+## Step 4/4. Apply ephemeral labels using `server_info`
+
+<Notice type="note">
+   Applying ephemeral labels is only supported for servers.
+</Notice>
+
+You can apply ephemeral labels to a Teleport instance by creating a `server_info`
+resource for the instance. For a server with name `<name>`, its matching `server_info`
+should be named `si-<name>`.
+
+To add ephemeral labels:
+
+1. Create the file `server_info.yaml` and paste the following into it:
+
+   ```yaml
+   # server_info.yaml
+   kind: server_info
+   metadata:
+      # Replace "example-node" with the name of the node to apply labels to
+      name: si-example-node
+   spec:
+      new_labels:
+         "foo": "bar"
+   ```
+
+   Run the following to create the `server_info` resource:
+
+   ```code
+   $ tctl create server_info.yaml
+   ```
+
+1. Verify that you have added the label by running the following command on your local
+computer. Your Teleport user must be authorized to access the server. Teleport
```

Comment:
So anyone who can access a server can update ephemeral labels? Is this concerning?

Reply:
You only need access to the server to verify that the labels were added.